openapi/components/schemas/Organizations/OrganizationSettings.yaml

```
@@ -38,3 +37,22 @@ properties:
       Available only for US-based organizations.
     items:
       $ref: ./TaxLocation.yaml
+  notifications:
+    description: Organization access-related notification settings.
+    type: object
+    properties:
+      notifyOnUserAccessChanges:
+        description: Whether to send notification when users are created and updated.
+        type: boolean
+        default: false
+      notifyOnApiKeyAccessChanges:
+        description: Whether to send notification when API keys are created and updated.
```

nitpick: [Boolean fields format](https://github.com/Rebilly/api-definitions/blob/main/WRITING-STYLE.md#boolean-fields)
```suggestion
        description: Specifies whether to send notifications when users are created or updated.
        type: boolean
        default: false
      notifyOnApiKeyAccessChanges:
        description: Specifies whether to send notifications when API keys are created or updated.
```